stories/dooboo-ui/Select.stories.tsx

```diff
@@ -0,0 +1,278 @@
+import { Image, View } from 'react-native';
+import { Item, Select } from '../../main/Select';
+import React, { ReactElement } from 'react';
+
+import { ContainerDeco } from '../../storybook/decorators';
+import LoadingIndicator from '../../main/LoadingIndicator';
+import { storiesOf } from '@storybook/react-native';
+import styled from 'styled-components/native';
+
+const CustomContainer = styled.View` 
+  width: 100%;
+  height: 100%;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+`;
+const TogglerWrapper = styled.View`
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: 10px;
+`;
+const ThemeSwitch = styled.Switch`
+  width: 30px;
+  height: 30px;
+`;
+const CustomTextInput = styled.TextInput`
+  background-color: #ffffff;
+  width: 190px;
+  height: 50px;
+  padding: 10px;
+  font-size: 12px;
+`;
+const CustomSelectContainer = styled.View`
+  width: 185px;
+  height: 45px;
+  align-items: center;
+  justify-content: center;
+`;
+const CustomText = styled.Text`
+  font-size: 10px;
+  font-weight: bold;
+  margin: 5px;
+`;
+
+const Default = (): React.ReactElement => {
+  // [Select] states
+  const [selectedValue, onSelectValue] = React.useState<
+    string | number
+  >();
+  const [isOpen, toggleSelect] = React.useState<
+    boolean
+  >(false);
+
+  return (
+    <CustomContainer style={{ justifyContent: 'center' }}>
+      <CustomSelectContainer>
+        <Select
+          opened={isOpen}
+          loading={false}
+          disabled={false}
+          showArrow={true}
+          isDarkMode={false}
+          bordered={true}
+          activeOpacity={0.5}
+          placeholder={'New fancy select'}
+          value={selectedValue}
+          onSelect={(value): void => onSelectValue(value)}
+          onOpen={(isOpen): void => toggleSelect(isOpen)}>
+          <Item value={'Item-1'}>{'Item-1'}</Item>
+          <Item value={'Item-2'}>{'Item-2'}</Item>
+          <Item value={'Item-3'}>{'Item-3'}</Item>
+        </Select>
+      </CustomSelectContainer>
+    </CustomContainer>
+  );
+};
+
+const ChangeProps = (): React.ReactElement => {
+  const [darkTheme, setDarkTheme] = React.useState<boolean>(true);
+  const [borderless, setBorder] = React.useState<boolean>(true);
+  const [showArrow, setShowArrow] = React.useState<boolean>(true);
+  const [disabled, setDisabled] = React.useState<boolean>(false);
+
+  // [Select] states
```

Comment:
This comment can be removed since it doesn't give any detail.

Try to give comments only if you want to refer to some additional information. Such as why do we need such variables? Where can we see the additional story (Something like links to github issues)?